### ISSUE TITLE
Goober styles leak into global scope

### ISSUE THREAD

**Issue by jan-wilhelm** [harshzalavadiya/react-multi-select-component#59]
Not sure if this is an issue with this repo or with Goober.

The [styles that are defined for the `DefaultRenderer` in select-panel/default-item.tsx](https://github.com/harshzalavadiya/react-multi-select-component/blob/99b13b1bb0760baaa64f200369b41542abca7faf/src/select-panel/default-item.tsx#L14-L24) leak out into the global CSS scope.

This source code:
```javascript
const DefaultRenderer = css({
  "input,span": {
    verticalAlign: "middle",
    margin: 0,
  },
  span: {
    display: "inline-block",
    paddingLeft: "5px",
  },
  "&.disabled": {
    opacity: 0.5,
  },
});
```

creates this CSS when rendered with Goober:
```css
.go3532419405 input,span{vertical-align:middle;margin:0;}
.go3532419405 span{display:inline-block;padding-left:5px;}
.go3532419405.disabled{opacity:0.5;}
```

The `,span` is the problematic part that leaks out into the global styles. The browser interprets that first line as "all elements that
- match that goober class and also are an input tag, or
- are a span tag"

<img width="264" alt="Screenshot of the problematic style tag in the DOM" src="https://user-images.githubusercontent.com/16737505/86117111-c20eca80-bace-11ea-9a71-12a4d0bff09d.png">

I am not sure how to fix this, perhaps this is an issue with Goober itself.

**Comment by harshzalavadiya** [comment]
@jan-wilhelm 

Thanks for pointing it out,
Style leak is fixed now please update your package to `v2.0.12+`

**Comment by jan-wilhelm** [comment]
Awesome, thank you very much for the quick response and for this fantastic package!!
